recipes/catch2/2.x.x/test_package/conanfile.py

```diff
@@ -2,40 +2,39 @@
 from conan.tools.cmake import CMake, CMakeToolchain
 from conan.tools.build import can_run
 from conan.tools.cmake import cmake_layout
+from conan.tools.files import save, load
 import os
-import yaml
+import json
+
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "CMakeDeps", "VirtualRunEnv"
     test_type = "explicit"
 
-    _tests_todo = []
 
     @property
     def _todos_filename(self):
-        return os.path.join(self.recipe_folder, self.folders.generators, "catch2_test_to_do.yml")
+        return os.path.join(self.build_folder, "catch2_test_to_do.yml")
```

Comment:
Seems wrong to put this in the build_folder... the output from the generate method should be in the generators folder 😖 

How does the `CMakeToolchian` do this? (Obviously it does work in both 1.x and 2.0 since the code is different) But this is counter intuative